wger/settings_global.py

```diff
@@ -199,9 +204,19 @@
 #
 # Login
 #
-LOGIN_URL = '/user/login'
-LOGIN_REDIRECT_URL = '/'
 
+LOGIN_URL = '/'
+LOGIN_REDIRECT_URL = '/dashboard'
+SOCIAL_AUTH_NEW_USER_REDIRECT_URL = '/dashboard'
+SOCIAL_AUTH_LOGIN_REDIRECT_URL = '/dashboard'
+
+SOCIAL_AUTH_FACEBOOK_KEY = "116172672420060"
```

Comment:
look into how you can add these secret keys as env variables.
They are supposed to be secret and should not appear on your GitHub repo